### PR TITLE
Add github action for build/push docker image

### PR DESCRIPTION
**Pull request by wazsone**
Add github action for build/push docker image

Signed-off-by: Ruslan Bayandinov <ruslan.bayandinov@xored.com>

### CODE REVIEW

File: .github/workflows/docker-build-push.yml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+on:
+  push:
+    branches: [ develop-bridge ]
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+env:
+  IMAGE_NAME: xored-mosquitto
```
Comment (kisel) [inline .github/workflows/docker-build-push.yml]: unused

File: .github/workflows/docker-build-push.yml [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+      id: buildx
+      uses: docker/setup-buildx-action@v1
+
+    - name: Log into registry
+      run: echo "${{ secrets.GITHUB_TOKEN }}" | docker login ghcr.io -u ${{ github.actor }} --password-stdin
+
+    - name: Build and push docker image
+      run: |
+        IMAGE_ID=ghcr.io/${{ github.repository }}
+        VERSION=latest
```
Comment (kisel) [inline .github/workflows/docker-build-push.yml]: git tags should be supported in order to produce immutable docker image tags

**Review by kisel — APPROVED** [review]
lgtm, but let's add tags support

File: .github/workflows/docker-build-push.yml (hunk 1 above)
Comment (wazsone) [reply .github/workflows/docker-build-push.yml]: fixed

File: .github/workflows/docker-build-push.yml (hunk 2 above)
Comment (wazsone) [reply .github/workflows/docker-build-push.yml]: fixed

**Pull request merged**